fluent.syntax/fluent/syntax/errors.py

```diff
@@ -60,7 +60,7 @@ def get_error_message(code, args):
     if code == 'E0025':
         return 'Unknown escape sequence: \\{}.'.format(args[0])
     if code == 'E0026':
-        return 'Invalid Unicode escape sequence: {}'.format(args[0])
+        return 'Invalid Unicode escape sequence: {}.'.format(args[0])
```

Comment:
This was a difference to js, which triggered structure failures on ex-behavior tests.